citizennet/angular-schema-form
# API-2905 Fix "Required" Not Taking Precedence

## Jira Ticket

- [https://citizennet.atlassian.net/browse/API-2905](https://citizennet.atlassian.net/browse/API-2905)

## What does this pull request do?

This PR addresses an issue where the `Required` validation text fails to take precedence for array items in the form. I had done a lot of click testing to verify this fix doesn't bring any unintended side effects. Non-`Required` validation errors will still occur, as intended.

## How should this be manually tested?

1. Begin building a new request: [https://jh.aws.citizennet.com/manager/#/requests/new/details](https://jh.aws.citizennet.com/manager/#/requests/new/details)
2. Select a request group that utilizes the `Conde Nast Entertainment` request template.
3. Navigate to the creative tab to trigger form validation: `Creative Post Build (External)`
4. Navigate back to the details tab: `Adbook Order Details (Internal)`
5. Verify the `Brands` field/dropdown displays the following text in red: `Required`
